src/components/Proposals/ProposalPage/CastVoteDialog/CastVoteDialog.tsx

```diff
@@ -46,6 +58,19 @@ function BasicVoteDialog({
 
   const vpToDisplay = getVpToDisplay(votingPower, missingVote);
 
+  useEffect(() => {
```

Comment:
I am not a fan of adding this to the vote dialog because we would need to replicate it across various contexts, such as the gasless voting dialog, SCW voting dialog, and others.

Reply:
Yea, I agree. We are already having to duplicate it twice now (advanced voting, and regular voting).
Is you suggestion more product driven (let's prompt them in a different place), or tech driven (let's rethink how we are triggering the dialog post vote)?